src/foam/nanos/client/ClientBuilder.js

```diff
@@ -9,18 +9,19 @@ foam.CLASS({
   name: 'ClientBuilder',
 
   implements: [
-    'foam.box.Context',
     'foam.mlang.Expressions'
   ],
 
   requires: [
     'foam.box.HTTPBox',
     'foam.box.RetryBox',
-    'foam.dao.ClientDAO',
+    'foam.dao.EasyDAO',
     'foam.dao.RequestResponseClientDAO',
-    'foam.nanos.boot.NSpec'
+    'foam.nanos.boot.NSpec',
   ],
 
+  axioms: [ foam.pattern.Singleton.create() ],
```

Comment:
Why singleton?

Reply:
Hmm I guess it does prevent us from being able to build clients from different servers right? This class doesn't really allow for that because the nspecDAO url is hardcoded but that's easy to make configurable if we want to do that.  I'll make it not a singleton.